alerting/src/main/kotlin/org/opensearch/alerting/model/Finding.kt

```diff
@@ -0,0 +1,202 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ *   Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package org.opensearch.alerting.model
+
+import org.opensearch.alerting.elasticapi.instant
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import org.opensearch.common.io.stream.Writeable
+import org.opensearch.common.lucene.uid.Versions
+import org.opensearch.common.xcontent.ToXContent
+import org.opensearch.common.xcontent.XContentBuilder
+import org.opensearch.common.xcontent.XContentParser
+import org.opensearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import org.opensearch.commons.authuser.User
+import java.io.IOException
+import java.time.Instant
+
+class Finding(
+    val id: String = NO_ID,
+    val logEventId: String = NO_ID,
+    val monitorId: String,
+    val monitorName: String,
+    val monitorUser: User?,
+    val monitorVersion: Long = NO_VERSION,
```

Comment:
Do we need the `monitorUser` information? This index would be accessible through the discover dashboard page right? Based on that, I dont see what benefit we can get from this.
For the `monitorVersion`, I dont think this will be needed.

Reply:
That's good feedback. I'll remove `monitorUser` and `monitorVersion`.